### ISSUE TITLE
Input preparation question

### ISSUE THREAD

**Issue by ceposadap** [jbpease/dfoil#4]
Dear James,

Will you help me out with some questions? I want to evaluate introgression among 4 taxa and an outgroup. For this analysis, I began with the complete genome sequences of many individuals of each of the taxa that I want to study. I mapped each genome to the reference and, in the end, I ended up with a gvcf file per genome. What I don't understant is how to transition from this to an appropriate input for d-foil.

Should I combine all gvcfs from each taxa, so that I end up with 5 gvcfs? Or should I combine them all, or not combine them at all? After this, I must convert this gvcfs to the fasta format, right?

Thanks for your time!

Carlos Posada

**Comment by jbpease** [comment]
Dear Carolos,

I would recommend checking out my other software package MVFtools (https://github.com/jbpease/mvftools).  You can convert VCF files to an MVF (which just has the allele data) and either compute D-statistics directly, or use the PatternCount function to generate the input files for dfoil.  Let me know if I can help more.

James

**Comment by ceposadap** [comment]
Dear James,

Thank you for your answer. I checked out your package MVFtools but I
encountered another problem. I have many samples that I worked on from
mapping to calling haplotypes with GATK's HaplotypeCaller, which in the end
produced a VCF file per sample. I then used GATK's GenotypeGVCFs to combine
and genotype several samples from the same taxon, and I obtained one
genotyped VCF file per taxon. Finally, I used bcftools to merge this
different VCF files into one.

MVFtools works on a normal VCF file and creates an MVF. However, it doesn't
work when I try to convert the combined and genotyped VCF files. Does
MVFtools require that the VCF file comes from a single sample? I'm trying
to use d-foil for detection of introgression using five taxa, but I haven't
figured out how to fix the input for the program.

Best,
Carlos

2018-06-22 14:11 GMT-05:00 James Pease <notifications@github.com>:
